base/complex.jl

```diff
@@ -1,5 +1,13 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+    Complex{T<:Real}
```

Comment:
Yes, I was thinking about that. We can try to incorporate your `show_supertypes()` function in these docstrings maybe?